### DISCUSSION TITLE
Split issue

### DISCUSSION THREAD

**Discussion by iamLazyCode** [mifi/lossless-cut#2125] [General]
i am trying to split a video

as in this screenshot u can see that at this time frame i have a different scene `from where i actually wanted to cut` 

![image](https://github.com/user-attachments/assets/daab2681-804a-4c94-914a-4e45ad52dcae)

but when i export the video it started from here , which is few seconds before

![image](https://github.com/user-attachments/assets/4e50a8b0-9224-4cf5-8b9a-0c4e878f0907)

can anyone please me fix this...i am going nuts

**Comment by mifi** [comment]
https://github.com/mifi/lossless-cut/blob/master/issues.md#cutting-times-are-not-accurate
